internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```diff
@@ -305,4 +312,11 @@ EnvironmentManagerRole:
             - 'cloudformation:DescribeStacks'
             - 'cloudformation:DeleteStack'
           Resource:
-            - !Sub 'arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${AWS::StackName}/*'
\ No newline at end of file
+            - !Sub 'arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${AWS::StackName}/*'
+        - Sid: RDS
+          Effect: Allow
+          Action:
+            - 'rds:DescribeDBInstances'
+            - 'rds:DescribeDBClusters'
+            - 'rds:DescribeDBProxies'
```

Comment:
Why are `rds:DescribeDBClusters` and `rds:DescribeDBProxies` required?

Reply:
my thinking is that it actually probably makes sense for us to include logic to inject these hosts (from `Clusters` and `Proxies`), but i think it can wait until a future release. however, if we include permissions now, we won't have to add another env version check 😅

Reply:
i actually wrote the logic to include clusters, and removed proxies!